src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -104,6 +104,8 @@ public struct HostAnalyticsInfo
         public string ParentId;
         /// Dynamo host session id for analytics purpose.
         public string SessionId;
+        /// AppVersion provided by host for analytics purpose.
+        public string AppVersion;
```

Comment:
could we reconsider this name and provide a better comment. IE how should anyone figure out what the difference is between hostinfo and app version?

Reply:
1. IF there is no difference in there purpose why do they both still exist?
2. For example, what does `App` refer to? What App? What Version, Version of Dynamo? The Host App? etc. If I just saw this with no example I would have no idea what to set it to.

Reply:
DynamoModel static AppVersion served as default - but since we only used it for analytics initialization I removed from here per your other comment.